manifests/proxydhcp.pp

```diff
@@ -49,6 +49,12 @@
     failover       => $failover,
   }
 
+  exec { 'setfacl_dhcp':
+    command => 'setfacl -R -m u:theforeman:rwx /etc/dhcp /var/lib/dhcpd',
```

Comment:
I believe this is should be ```u:${::foreman_proxy::user}:rx```. First of all this automatically uses the correct user and secondly I think the proxy only reads files. All changes happen through ```omshell``` but maybe then the user also needs to write access in ```/var/lib/dhcpd```. Does it also need to be recursive?

Another thing to consider is that the acl package might not be installed.